grant/projects/helix.md

```diff
@@ -0,0 +1,97 @@
+
+- Locking issuance and redemption of assets RING & KTON
+- User-friendly transaction interface
+- Transaction history indexing and display
+
+### **Helix LnBridge V1.0 (Already Launched)**
+This version is live between Darwinia and Ethereum, reducing user cross-chain costs and improving transaction speed. Security has been enhanced to a certain extent. This version would be upgrade to V3.0 near future.
+
+- Introducing liquidity to reduce reliance on asset issuance solution.
+- Safer and faster.
+
+### **Helix LnBridge V2.0 (Already Launched)**
+This version is live between Darwinia and Arbitrum, further lowering user cross-chain costs through a collateral mechanism. It connects Darwinia to the Ethereum Layer 2 network.
+
+- The collateral is introduced in this version.
+- Less use of generic messages.
+
+### **Helix LnBridge V3.0 (Not Launched Yet)**
+This version enhances user cross-chain transaction depth by removing the collateral mechanism. This update is designed to streamline the cross-chain experience and reduce relayer pledging costs.
+
+- Introduced Penalty reserve to reduce relayer's pledge costs and further enhance its own security.
+
+## **Future Development Roadmap**
+The following milestones are not included in the collaboration grant assessment and are included to communicate Helix Bridge team's vision and direction.
+
+- Helix 3.0: represents a comprehensive solution that integrates xToken, LnBridge v2.0, and LnBridge v3.0, running concurrently and complementing each other.
+- Upgrade xToken solution to be more generic and adaptable and ensure compatibility with various networks.
+- Achieve full interconnection of connected networks, reducing user asset cross-chain path lengths.
+- Helix Dao
+- Asset monitoring, user data analysis
+
+## **Collaboration Funding**
+Total funding amount: 3 ETH and 9,000,000 RING.
+
+The first fund, paid in ETH, is dedicated to supporting Helix Bridge's network deployment expenses. With this funding, Helix aims to maximize cross-chain interconnectivity, minimizing the pathways to link the Darwinia Network with other ecosystems. The primary objective is to establish as many connections as possible, ensuring a robust and comprehensive network that facilitates seamless asset transfer.The second fund, paid in RING, will be utilized to provide liquidity support for RING assets. Helix Bridge commits not to expend this fund but rather engage in long-term staking to offer liquidity. The focus is on sustaining a continuous and stable provision of liquidity, contributing to the overall health and functionality of the Helix Bridge ecosystem.
+
+The 1st payment date: After Helix completes the Darwinia to Ethereum xToken business upgrade PR merge, with the address being 0x88a39B052d477CfdE47600a7C9950a441Ce61cb4 on Arbitrum or Ethereum.
```

Comment:
@xiaoch05 Would you please check this address, better to use the helix dao address

Reply:
https://github.com/darwinia-network/collaboration/pull/33/commits/52830a8549c49779663c2d677410aa5fd6612fd9
updated to the helix dao address.